docs/aws/audit/cloudformationmonitoring/rules/enable_stack_termination_protection_remediation.mdx

```diff
@@ -1,6 +1,214 @@
+
+4. **Run the Script**:
+   - Execute the script to create the CloudFormation stack with termination protection enabled.
+
+By following these steps, you can ensure that any CloudFormation stack created using this script will have termination protection enabled, preventing accidental deletions.
+</Accordion>
+
+</AccordionGroup>
+</Tab>
+<Tab title='Cause'>
+### Check Cause
+<AccordionGroup>
+<Accordion title='Using Console' defaultOpen='true'>
+1. Sign in to the AWS Management Console.
+2. Navigate to the CloudFormation service. You can do this by typing "CloudFormation" into the search bar and selecting it from the dropdown menu.
+3. In the CloudFormation console, you will see a list of your stacks. Select the stack you want to check.
+4. In the stack details pane, look for the "Termination Protection" field. If it is set to "Enabled", then termination protection is on. If it is set to "Disabled", then termination protection is off.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. Install and configure AWS CLI: Before you can start using AWS CLI, you need to install it on your local machine and configure it with your AWS account credentials. You can do this by running the following commands:
+
+   ```
+   pip install awscli
+   aws configure
+   ```
+   You will be prompted to provide your AWS Access Key ID, Secret Access Key, Default region name, and Default output format.
+
+2. List all CloudFormation stacks: Use the following AWS CLI command to list all the CloudFormation stacks in your AWS account:
+
+   ```
+   aws cloudformation list-stacks --stack-status-filter CREATE_COMPLETE ROLLBACK_COMPLETE UPDATE_COMPLETE UPDATE_ROLLBACK_COMPLETE
+   ```
+   This command will return a list of all CloudFormation stacks that are in a completed state.
+
+3. Describe each stack: For each stack returned by the previous command, use the following AWS CLI command to get detailed information about the stack:
+
+   ```
+   aws cloudformation describe-stacks --stack-name <StackName>
+   ```
+   Replace `<StackName>` with the name of the stack. This command will return a JSON object that contains detailed information about the stack.
+
+4. Check termination protection status: In the JSON object returned by the previous command, look for the `EnableTerminationProtection` field. If the value of this field is `false`, then termination protection is not enabled for the stack. If the value is `true`, then termination protection is enabled.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install and configure AWS SDK for Python (Boto3) on your local system. This will allow you to interact with AWS services using Python.
+
+```python
+pip install boto3
+aws configure
+```
+
+2. Import the necessary modules and create a session using your AWS credentials.
+
+```python
+import boto3
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>